data.json

```diff
@@ -2,6 +2,7 @@
   "Go": {
     "DefaultVersion": "go1.20.14",
```

Comment:
(not a request for this PR, since unrelated)

The buildpack default Go is v1.20, which if I'm reading the version policy is unsupported and thus potentially insecure?
https://go.dev/doc/devel/release#policy